### ISSUE TITLE
Cannot select Skill Endpoint and bot throw error 'Value cannot be null. (Parameter ‘uriString’)' 

### ISSUE THREAD

**Issue by aiipe** [microsoft/BotFramework-Composer#6304]
## Describe the bug

Added a remote skill bot, but from the trigger cannot select the Skill Endpoint and bot throw error 'Value cannot be null. (Parameter ‘uriString’)' 

## Version

v1.3.1

## OS
- [ ] Windows

## To Reproduce

1. Add remote skill
![image](https://user-images.githubusercontent.com/4598977/110248833-f3b92900-7fad-11eb-8c77-4db2398a12ff.png)

2. Create a trigger and connect to remote skill. Skill Endpoint dropdown can see the ToDoSkill value, but unable to select.
![image](https://user-images.githubusercontent.com/4598977/110248879-324ee380-7fae-11eb-8978-9aa9206f7a7e.png)

3. Skill manifest as below
![image](https://user-images.githubusercontent.com/4598977/110249057-139d1c80-7faf-11eb-9e12-2af8a1fe4224.png)

4. Parent bot setting as below
![image](https://user-images.githubusercontent.com/4598977/110249099-5a8b1200-7faf-11eb-8fb8-b13b96cb37fc.png)
 
5. Start the parent bot and it throw error '**Value cannot be null. (Parameter 'uriString')**' when connecting to skill bot 

## Expected behavior

Bot will reply message from skill bot

**Comment by stevkan** [comment]
@aiipe, at this moment, I am unable to repro the issue you are experiencing. Have you tried with a new Composer project, new skill & manifest? Does this occur in multiple projects or just this one?

One thing I did notice is the schema value listed in your manifest is different than what is generated when I create a manifest. Can you try updating to the following and see if that makes a difference? I'm not expecting it to, necessarily. But, it's worth trying.

`"schema": "https://schemas.botframework.com/schemas/skills/v2.1/skill-manifest.json"`

**Comment by stevkan** [comment]
@aiipe, I inadvertently triggered the error you have been receiving. In my case, I had removed the remote skill however the "Connect to a skill" dialog event was still present. When I re-added the remote skill I had failed to go back into the "Connect to a skill" dialog event and re-assign the "Skill Endpoint" field. The "Skill Dialog Name" field was populated, but the "Skill Endpoint field was not. When I ran the bot and tested calling the skill, it returned the error.

In addition to my previous post, with the remote skill setup, can you go back into the "Connect to a skill" dialog event and verify that all the necessary fields are populated and try, again? I know you posted that as one of your steps, but it's worth double checking.

I'm going to assume that you are restarting your bot each time following any changes. 🙂

**Comment by aiipe** [comment]
hi @stevkan , i updated the skill bot schema as per your suggestion. Now I tried create a new bot and add remote skill, I had success connect to remote skill bot as expected. But after that I adding few dialog and triggers, then suddenly the 'Skill Endpoint' field value gone eventhrough I didnt do any changes to that 'Connect to a skill' dialog event. I did restarting my bot for each time changes, but now i unable to select the 'Skill Endpoint' field again.
